### DISCUSSION TITLE
RowSpan Table @itwin/itwinui-react

### DISCUSSION THREAD

**Discussion by Paolo-lab** [iTwin/iTwinUI#1280] [Q&A] [answered]
Hello good morning, I should create a table within the viewer that groups rows together, there is the possibility of using the rowspan with the

control
I am attaching an example in xls that I should emulate with the

control
Thanks for the precious help
<img width="364" alt="RowSpan" src="https://github.com/iTwin/iTwinUI/assets/57446652/4a0fe89f-196d-407f-8784-53446bd8e3fd">

**Comment by mayank99** [comment, marked as answer]
hey @Paolo-lab, our Table does not support rowspan. The closest you can get is using "expandable subrows" (using `subRows` property in data) or "custom cells" (using `Cell`/`cellRenderer` properties in columns).
